mushr_utils/install/mushr_install.bash

```diff
@@ -102,15 +119,21 @@ cd $MUSHR_WS_PATH/catkin_ws/src/ && vcs import < mushr/base-repos.yaml && vcs im
 
 # Make custom mushr_noetic script
 cat <<- EOF > ${MUSHR_INSTALL_PATH}/mushr_noetic
+#!/bin/bash
 export MUSHR_INSTALL_PATH=${MUSHR_INSTALL_PATH}
 export MUSHR_REAL_ROBOT=${MUSHR_REAL_ROBOT}
 export MUSHR_WS_PATH=${MUSHR_WS_PATH}
 export MUSHR_COMPOSE_FILE=${MUSHR_COMPOSE_FILE}
 export MUSHR_OS_TYPE=${MUSHR_OS_TYPE}
-if [ \$# == 0 ] || [ \$1 == "run" ];
+if [[ \$# == 0 ]] || [[ \$1 == "run" ]];
 then
-  docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash
-elif [ \$1 == "build" ];
+  if [[ \$# == 2 ]];
+  then
+    docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash -ic "\${2}"
+  else
+    docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash
+  fi
+elif [[ \$1 == "build" ]];
```

Comment:
do we need this additional elif? could the following lines not be put in the prior conditional?

Reply:
See the comment above

Reply:
my b, misread it